packages/storefront-lwc/src/modules/commerce/basket/basket.js

```diff
@@ -61,6 +61,15 @@ export default class Basket extends LightningElement {
             .catch(error => {
                 console.log('error received ', error);
             });
+
+        window.dispatchEvent(
```

Comment:
@blittle : Thank you for your feedback.  We were not able to use component dispatch event before thus @coopaq suggested to use window dispatch event.  Since your lwce router PR just got merged, I can make the change to dispatch event from component. Thx!